hotham/src/systems/rendering.rs

```diff
@@ -154,11 +154,7 @@ pub fn rendering_system(
                 let instance = &instanced_primitive.instances[cull_result.index_instance as usize];
                 let draw_data = DrawData {
                     global_from_local: instance.global_from_local,
-                    inverse_transpose: instance
-                        .global_from_local
-                        .try_inverse()
-                        .unwrap()
-                        .transpose(),
+                    local_from_global: instance.global_from_local.try_inverse().unwrap(),
```

Comment:
I don't think this is correct! Everything I've read about these normals indicates they need a transpose. 😨

Reply:
I'm applying it from the left, treating the normal as a row vector. See here: https://github.com/leetvr/hotham/pull/330/files#diff-02130a77bf74f1b203443bd19cca29f2adcc1dd8d7b991bc2b83f39d1589c976R32

Reply:
This is arguably the more correct way of treating normals because they are bivectors. Mathematical bike shedding perhaps 😛

Reply:
So as a mathematical simpleton, why do other implementations use the inverse transpose, when what you're suggesting is far simpler?

Reply:
Maybe because it's hard to make math simple? Bivectors are not commonly taught and this can be solved without them. Few people are curious enough to keep looking for better ways to express math once they have something that works.